.github/fabricbot.json

```diff
@@ -634,6 +634,85 @@
         ]
       }
     },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "IssueCommentResponder",
+      "version": "1.0",
+      "id": "bvMkigH2HPgUniYFCNco9",
+      "config": {
+        "conditions": {
+          "operator": "and",
+          "operands": [
+            {
+              "name": "commentContains",
+              "parameters": {
+                "isRegex": true,
+                "commentPattern": "\\/feedbackHub"
+              }
+            },
+            {
+              "operator": "or",
+              "operands": [
+                {
+                  "name": "activitySenderHasAssociation",
+                  "parameters": {
+                    "association": "OWNER"
+                  }
+                },
+                {
+                  "name": "activitySenderHasAssociation",
+                  "parameters": {
+                    "association": "MEMBER"
+                  }
+                },
+                {
+                  "name": "activitySenderHasAssociation",
+                  "parameters": {
+                    "association": "COLLABORATOR"
+                  }
+                }
+              ]
+            }
+          ]
+        },
+        "eventType": "issue",
+        "eventNames": [
+          "issue_comment"
+        ],
+        "taskName": "Helper to mark as feedback hub",
+        "actions": [
+          {
+            "name": "addReply",
+            "parameters": {
+              "comment": "Hi! We've identified this issue that best is suited for Windows Feedback Hub.  To do this on Windows, WinKey+F will bring it up!"
+            }
+          },
+          {
+            "name": "closeIssue",
+            "parameters": {}
```

Comment:
I had a look for the api the other day, but haven't found it (yet). I wonder what parameter(s) could be possible here. Maybe the "close as.."?

Reply:
asked the team, this param doesn't seem to be documented